spec/integration/drift_integration_spec.rb

```diff
@@ -13,6 +13,6 @@
 
   it 'embeds the script with account_id' do
     expect(page.find('script')).to have_content('js.driftt.com')
-    expect(page.find('script')).to have_content('DRIFT_ID')
+    expect(page.find('script')).to have_content('\'DRIFT_ID\'')
```

Comment:
maybe we check for the whole `drift.load('DRIFT_ID');` to be sure we actually check for that and not another DRIFT_ID?

Reply:
Was in a meeting, but have updated the check now